### DISCUSSION TITLE
add target option for measurable habit

### DISCUSSION THREAD

**Discussion by wr3e** [iSoron/uhabits#1275] [Feature Requests]
There are cases that we need to try not to reach a limit, or we need to complete in a range, for ex., we perform habit A which is related with health, we need to finish 30 to 50 a day, it's no use if we only finish 29, and it'll be excessive if we do 51, so I need the target just between 30 and 50, <30 or >50 both will impact the habit performance. Hope the target give us the option "between", "less than", "at least" instead of a fixed target

**Comment by hiqua** [comment]
Do you have an example?

The point of habits is to adopt "good" behaviors that are not easy to get into. You seem to describe something which would be hard to do 30 of, but once you've reached that threshold you could tend to overdo it beyond 50? An example would definitely help.

**Reply by wr3e** [reply]
Thank you, we'll, isn't >30 and <50 already an example. The idea I bring up here is, "a target is good, but too much of a good thing can be bad"

**Reply by hiqua** [reply]
I meant an example of a habit where it would make sense, not an example of targets?

**Reply by wr3e** [reply]
The idea is already the best sense. 
This app is total free, so even myself feel that I am ungrateful or critical if I show any dissatisfied. 
I don't know what's the difficult of this feature request, I don't understand why you can't think from the user's perspective, I read your reply to requests and suggestions, in many cases, your idea are not to consider users' needs and requests but ask them to accept what the app is, then what's the point to give this "feature request"? Actually before I post this request, I have thought what the reply will be and it's turned to be what I supposed. 
I don't mean to offend you, please don't let my words impact your mood, thank you for your time to reply me, forget this request, sincerely wish you a happy day.
I don't know how to close this request but it can be closed

**Reply by wr3e** [reply]
Again, please don't think I'm ungrateful and please don't feel offended, I'm sorry if you are.
I don't find where can delete or close this request, so I'll leave it open but I won't follow it.
Again, wish you a nice day

**Reply by hiqua** [reply]
I think we have to backtrack here a little bit, I'm not questioning your feature request, I'm just trying to understand in which cases this feature would be useful, and I don't have an example of an habit where it would bring something, do you have one? If you take a habit `run` more than `10km`, it makes sense to have this minimum, but it's easy to stop before your maximum so it doesn't bring much to have it in the app, right?
